src/index.js

```diff
@@ -272,6 +272,22 @@ const jsPrettier = (tree, {rules: {indent, eol}, jsBeautifyOptions}) => {
   return prettier(tree);
 };
 
+const addLang = (tree, {rules: {lang}}) => {
+  if (!lang) {
+    return tree;
+  }
+
+  tree.map(node => {
+    if (node.tag) {
+      node.attrs = Object.assign({}, {lang}, node.attrs);
```

Comment:
By the way there is an interesting method for working with attributes
https://github.com/posthtml/posthtml-beautify/blob/71aa6c2a70518d069f377f4880efb53ff67beb3e/src/index.js#L29-L35